src/components/TopBar.js

```diff
@@ -320,6 +320,7 @@ const TopBar = ({ showMenu = true, title, href, children }) => {
       role: 'banner',
       style: {
         ...styles.topBar,
+        backgroundColor: isTerra() ? colors.primary() : colors.ligth(), // Fallback color for a11y and if background images don't show
```

Comment:
This is a typo `ligth` should be `light`?

Reply:
🤦